crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,31 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
```

Comment:
the name of the method is not really clear to me in terms of what it returns, why not just call it `unreferenced_disks`?